src/switch_ivr_bridge.c

```diff
@@ -817,6 +817,8 @@ static void *audio_bridge_thread(switch_thread_t *thread, void *obj)
 #ifdef SWITCH_VIDEO_IN_THREADS
 		if (switch_channel_test_flag(chan_a, CF_VIDEO) && switch_channel_test_flag(chan_b, CF_VIDEO) && !vid_launch) {
 			vid_launch++;
+			vh.session_a = session_a;
```

Comment:
The reason why @damirn proposed this fix is that if the session_a somehow is deleted or destroyed then the video thread may access a garbage pointer. We switch to uuid so we can validate if session is still active by locating and locking it.